pkg/proc/arch.go

```diff
@@ -65,6 +66,11 @@ func (a *Arch) BreakpointInstruction() []byte {
 	return a.breakpointInstruction
 }
 
+// AltBreakpointInstruction returns an alternate encoding for the breakpoint instruction.
+func (a *Arch) AltBreakpointInstruction() []byte {
```

Comment:
Do you think it'd make sense to just change the original method to be `BreakpointInstructions` (plural) instead and just always return a slice?

Reply:
No strong opinions here. My reasoning was that BreakpointInstruction was the one we use and the other would be breakpoint instructions we might encounter.

Reply:
But changing to a slice is fine too.

Reply:
Ack, that makes sense to me.